lib/typescript/package.json

```diff
@@ -19,13 +19,13 @@
     "build": "parcel build index.ts"
   },
   "devDependencies": {
-    "@parcel/packager-ts": "2.4.1",
-    "@parcel/transformer-typescript-types": "2.4.1",
+    "@parcel/packager-ts": "^2.4.1",
+    "@parcel/transformer-typescript-types": "^2.7.0",
     "parcel": "^2.4.1",
     "typescript": ">=3.0.0"
   },
   "dependencies": {
-    "isomorphic-fetch": "^3.0.0"
+    "axios": "^0.27.0"
```

Comment:
This is a large library. Ideally we keep the SDK as lightweight as possible. Why introduce Axios instead of isomorphic-fetch?

Reply:
the bundle get formed but with 2 errors in using the typescript fetch code generator: 

```
@parcel/transformer-typescript-types: Accessors are only available when targeting ECMAScript 5 and higher.

  /Users/plockett/neeva Dropbox/Paul Lockett/Mac/Documents/GitHub/xyz/lib/typescript/runtime.ts:34:9
    33 | 
  > 34 |     set config(configuration: Configuration) {
  >    |         ^^^^^^^ Accessors are only available when targeting ECMAScript 5 and higher.
    35 |         this.configuration = configuration;
    36 |     }


@parcel/transformer-typescript-types: Class 'FetchError' incorrectly extends base class 'Error'.

  /Users/plockett/neeva Dropbox/Paul Lockett/Mac/Documents/GitHub/xyz/lib/typescript/runtime.ts:239:14
    238 | 
  > 239 | export class FetchError extends Error {
  >     |              ^^^^^^^^^^^ Class 'FetchError' incorrectly extends base class 'Error'.
    240 |     name: "FetchError" = "FetchError";
    241 |     constructor(public cause: unknown, msg?: string) {
```

Reply:
I was able to get it down to just the Fetch Error in #3